### ISSUE TITLE
Grafana KERBEROS authentication missing

### ISSUE THREAD

**Issue by mrvic** [GridProtectionAlliance/osisoftpi-grafana#25]
could be the same as #4 ????

# Plugin is missing KERBEROS :(
## We are not allowed to use BASIC auth
**Form should include** 
`WWW-Authenticate: Negotiate method`

![osi plugin missing kerberos](https://user-images.githubusercontent.com/6241361/49649837-a7778b00-fa2a-11e8-9fc2-43629f565646.PNG)

# Errors are shown in console from grafana:

![errors authorization](https://user-images.githubusercontent.com/6241361/49649846-a9d9e500-fa2a-11e8-807c-9427ca2da888.PNG)

# But if endpoints are reached directly trough browser they are working fine

![working directly trough browser](https://user-images.githubusercontent.com/6241361/49649851-ac3c3f00-fa2a-11e8-812e-1129e54959f1.PNG)

# example with REST test:
![rest test](https://user-images.githubusercontent.com/6241361/49650160-a430cf00-fa2b-11e8-8646-3a84430be3d1.PNG)

**Comment by Rohlik** [comment]
Kerberos support would be nice.

**Comment by jleimhofer** [comment]
Yes, I agree - would really appreciate the possibility to authenticate via Kerberos.

**Comment by sonicdoo** [comment]
Would really be great if there was support for kerberos auth!

**Comment by AndreasFridh** [comment]
+1
